CMakeLists.txt

```diff
@@ -668,6 +668,17 @@ if(BUILD_TOOLS AND BUILD_DX11 AND WIN32)
   set_property(DIRECTORY PROPERTY VS_STARTUP_PROJECT texconv)
 endif()
 
+#--- install .exes
+
+if(MINGW AND BUILD_TOOLS)
+    foreach(t IN LISTS TOOL_EXES ITEMS ${PROJECT_NAME})
```

Comment:
Remove the ``ITEMS ${PROJECT_NAME}`` since that install logic is already handled above.